lerna.json

```diff
@@ -2,5 +2,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "2.0.0-beta.7"
+  "version": "2.0.0"
```

Comment:
Why the change to version `2.0.0`? As far as I'm aware we haven't updated to this release. Also, isn't this PR for a D3 template? Why is this included?

Reply:
I'm also confused... I'll delete these parts then. I don't think I've touched these parts of the code.